packages/neo-one-website/docs/4-node/2-kubernetes.md

```diff
@@ -0,0 +1,137 @@
+---
+slug: node-kubernetes
+title: Kubernetes
+---
+
+In this section we will cover how to deploy a NEO•ONE Node to a kubernetes cluster.
+
+If you are unfamiliar with Kubernetes visit their [getting started](https://kubernetes.io/docs/tutorials/kubernetes-basics/) page,
+in particular we will be implementing a [stateful set](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/) locally using Kubernetes through Docker.
+
+---
+
+[[toc]]
+
+---
+
+## Requirements
+
+- [Docker](https://www.docker.com/get-started)
+  - Minimum: ***at least*** 2GB Memory, and 50GB Storage allocated
+  - Recommended: 4GB Memory, 60GB+ Storage allocated (you will need ~60GB of storage per pod)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  - You can enable kubectl through the docker GUI; Docker >> Preferences... >> Kubernetes >> Enable Kubernetes
+
+---
+
+## Getting Started
+
+The following deployment spec will create a statefulSet of `n` nodes defined by `spec.replicas`. Each requests 60GB of storage and 4GB of memory. If you do not have a default storage class set (docker will automatically create for local deployments) you will need to create one, see [storage classes](https://kubernetes.io/docs/concepts/storage/storage-classes/) for more information.
+
+We include a headless service named `neo-one-service`, this is a requirement of StatefulSets.
+
+```yml
+# node-spec.yml
+apiVersion: v1
+kind: Service
+metadata:
+  name: neo-one-service
+  labels:
+    app: neo-one-service
+spec:
+  ports:
+  - port: 1443
+    name: node
+  clusterIP: None
+  selector:
+    app: neo-one-node
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: node
+spec:
+  serviceName: "neo-one-node"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: neo-one-node
+  template:
+    metadata:
+      labels:
+        app: neo-one-node
+    spec:
+      containers:
+      - name: neo-one-node
+        image: quay.io/neoone/node
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 1443
```

Comment:
What is this port used for? Shouldn't this use whatever the default is for the rpc server?